### DISCUSSION TITLE
can I use a custom model like dreamlike

### DISCUSSION THREAD

**Discussion by dreammeai** [TheLastBen/fast-stable-diffusion#1286] [Q&A]
I tried using dreamlike but it keeps failing on my end to Trian. 

stable diffusion works 1.5 works fine except custom models. what seems to be the issue

**Comment by TheLastBen** [comment]
use this https://huggingface.co/spaces/anzorq/sd-to-diffusers to convert the model to diffusers, I'll fix the issue later.

**Reply by dreammeai** [reply]
the spaces won't open, after putting my token it does nothing for a long period of time.

**Reply by TheLastBen** [reply]
I'll fix the colab soon

**Reply by grexzen** [reply]
I am not sure if related, but I just updated the repo with your notebook after maybe 2 weeks, and all my models, vaes, embeddings in google drive are now deleted, nowhere to be found, including many merged models I made. 

Was that something you changed in the code to wipe out the current folders? Or to move the current folder to an archive? 

Thanks, just hoping I didn't lose all my stuff, had like 30 models in there.

**Reply by TheLastBen** [reply]
nothing in the code that wipes out storage, check the gdrive trash, everything should be there if you didn't manually empty the trash
